daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -94,6 +94,7 @@ message NodeExercise {
     com.daml.lf.value.VersionedValue return_value = 12;
     com.daml.lf.value.VersionedValue contract_key = 13; // optional
     KeyWithMaintainers key_with_maintainers = 14; // optional
+    repeated string observers = 15;
```

Comment:
We should document that this need to be empty for whatever is the current version and below. We also need to agree if we want to call this `observers` or `informees`. cc @remyhaemmerle-da @bame-da @andreaslochbihler-da